Pages/ShowIndividualEvent.razor

```diff
@@ -150,26 +158,28 @@ else
     {
         var karmaContext = new KarmaContext();
         var eventEquipment = new List<string>();
-        foreach (var equipment in karmaContext.SpecialEquipment.Include(p => p.Owner))
-        {
-            foreach (var volunteer in karmaContext.Events.Include(p => p.Volunteers).Where(p => p.Id == Id).Select(p => p.Volunteers).SelectMany(p => p))
-            {
-                if (equipment.Owner.Id == volunteer.Id)
-                {
-                    eventEquipment.Add(equipment.Name);
-                }
-            }
-        }
-        Equipment = eventEquipment;
```

Comment:
Completely forgot to fix this part. But its fixed in a nicer way than I imagined.